static-analysis/assert-count-between.php.disabled

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Webmozart\Assert\StaticAnalysis\AssertString;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @param mixed $value
+ *
+ * @return array|\Countable
```

Comment:
@muglug I found that `@psalm-assert` does not work with union types: expected? Should I report this?

Reply:
As of 10 minutes ago, you can now use `@psalm-assert countable` - https://psalm.dev/r/e9a2641c58

Reply:
Noice :+1: